### PR TITLE
man: reword the description of "main conf file"

### LINKED ISSUE #18458: Misleading statement in the man page of logind.conf
In the man page for logind.conf one reads the following

> The main configuration file is read before any of the configuration directories, and has the lowest precedence; entries in a file in any configuration directory override entries in the single configuration file.

and then later on this

> Files in /etc/ are reserved for the local administrator, who may use this logic to override the configuration files installed by vendor packages.

These statements seem to be contradictory as written as far as `/etc/systemd/logind.conf` is concerned; this file is inside `/etc/` but it can not be used to override vendor configuration according to the first statement. The latter statement should instead be
> Files in /etc/systemd/logind.conf.d/ ...

systemd 246 (v246.7-2.fc33)

### PR DESCRIPTION
**Pull request by keszybz**
While not really "wrong", the text we had could use a little editing.

Fixes #18458.

### CODE REVIEW

File: man/standard-conf.xml [hunk 1]
```diff
@@ -41,33 +41,30 @@
...
-    files specify the same option, for options which accept just a single value, the entry in the file with
-    the lexicographically latest name takes precedence. For options which accept a list of values, entries
-    are collected as they occur in files sorted lexicographically.</para>
+    <para>In addition to the "main" configuration file, drop-in configuration snippets are read from
+    <filename>/usr/lib/systemd/*.conf.d/</filename>, <filename>/usr/local/lib/systemd/*.conf.d/</filename>,
+    and <filename>/etc/systemd/*.conf.d/</filename>. Those drop-ins have higher precedence and override the
+    main configuration file. Files in the <filename>*.conf.d/</filename> configuration subdirectories are
+    sorted by their filename in lexicographic order, regardless of in which of the subdirectories they
+    reside. When multiple files specify the same option, for options which accept just a single value, the
+    entry in the file sorted later takes precedence, and for options which accept a list of values, entries
```
Comment (bluca) [inline man/standard-conf.xml]: "sorted last" rather than later, maybe?

**Review by bluca — APPROVED** [review]
lgtm

File: man/standard-conf.xml (hunk 1 above)
Comment (keszybz) [reply man/standard-conf.xml]: I had "last" initially, but then I thought there might be more than one such file, and changed this. But there's only one file that is "last"… I'll change it back.

**Review by bluca — APPROVED** [review]
lgtm

**Pull request merged**